src/metabase/api/dashboard.clj

```diff
@@ -784,38 +785,49 @@
   "How many results to return when chain filtering"
   1000)
 
-(mu/defn ^:private mappings->field-ids :- [:maybe [:set ms/PositiveInt]]
+(mu/defn ^:private mappings->fields :- [:maybe [:set [:map
+                                                         [:field-id ms/PositiveInt]
+                                                         [:options [:maybe map?]]]]]
   [parameter-mappings :- [:maybe [:or [:set dashboard-card/ParamMapping] [:sequential dashboard-card/ParamMapping]]]]
   (set (for [{{:keys [card]} :dashcard :keys [target]} parameter-mappings
-             :let  [field-clause (params/param-target->field-clause target card)]
-             :when field-clause
+             :let  [ttag (params/param-target->template-tag target card)]
```

Comment:
I'm confused why this is a `template-tag`, if I have a parameter that connects to a field on a MBQL query, it's not a template tag.